app/templates/components/curriculum-inventory-verification-preview-table1.hbs

```diff
@@ -0,0 +1,29 @@
+<h4 data-test-title id="verification-preview-table1">{{t "general.table1ProgramExpectationsMappedToPcrs"}}</h4>
+<table>
+  <thead>
+    <tr>
+      <th>{{t "general.programExpectationsId"}}</th>
+      <th colspan="3">{{t "general.programExpectations"}}</th>
+      <th colspan="2">{{t "general.physicianCompetencyReferenceSet"}} ({{t "general.physicianCompetencyReferenceSetAbbr"}})</th>
+    </tr>
+  </thead>
+  <tbody>
+    {{#each @data as |row|}}
+      <tr>
+        <td>{{t "general.notApplicableAbbr"}}</td>
+        <td colspan="3">{{{row.title}}}</td>
+        <td colspan="2">
+          <ul>
+            {{#each row.pcrs as |pcrs|}}
+              <li>{{pcrs}}</li>
+            {{/each}}
+          </ul>
+        </td>
+      </tr>
+    {{/each}}
+  </tbody>
+</table>
+
+{{#if @tocId}}
```

Comment:
I'm not sure I understand this, why not a fixed ID and why an `if` is it ever not present?

Reply:
i suppose i could simplify this. the whole linking business makes me a bit uneasy, as it creates these inter-dependencies between these components that fall outside the usual component nesting strategy.

Reply:
refactored, i pulled those links up and out.